.gitignore

```diff
@@ -12,17 +12,18 @@ venv*
 **/__pycache__
 **/*.egg-info
 *.pyc
+
 .tox/
 .mypy_cache/
-
 .coverage

```

Comment:
The addition of extra newlines in the `.gitignore` file seems unnecessary and could be cleaned up for better readability.